.github/workflows/installer-macos.yml

```diff
@@ -0,0 +1,74 @@
+# Based on https://github.com/tlambert03/napari/blob/master/.github/workflows/make_bundle.yml
+on:
+  pull_request:
+  release:
+    types:
+      - created
+
+name: Create macOS app bundle and DMG
+
+jobs:
+  build:
+    name: macOS app bundle
+    runs-on: macos-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7.9
```

Comment:
Please change this to to have the Python version you'd like to see for the installer.

Reply:
I don't understand your request. I set this to 3.7.9, which is what I expect to see for the installer. I believe that is the version that Spyder is officially supporting, correct?

Reply:
I resolved the issue. It was related to the "shell: bash -l {0}" callout